internal/executor/instance/containerbackend/podman_linux.go

```diff
@@ -72,6 +76,15 @@ func NewPodman() (ContainerBackend, error) {
 		HTTPClient: podman.httpClient,
 	})
 
+	// Query server's version and activate bug workarounds (if applicable)
+	version, err := podman.SystemInfo(context.Background())
+	if err != nil {
+		return nil, err
+	}
+	if version.Version == "3.0.0" {
```

Comment:
Should it check if it's 3.0.0+?

Reply:
I hope this will get fixed soon, so only checking for the broken version here.